frontend/src/components/Home/Main.tsx

```diff
@@ -38,6 +38,16 @@ export const AppLayout: React.FC = () => {
   const { data: isAuthenticated, isPending: isPendingAuth } = useQuery(
     isAuthenticatedQuery()
   );
+  const pathname = useLocation({ select: (location) => location.pathname });
+
+  useEffect(() => {
+    // This fixes the issue of the main panel (e.g. settings, connections...) starting from the bottom
```

Comment:
Maybe change this docstring to not mention 'issue'. someone reading this in the future won't care about a past issue. just say that this makes sure the pages are scrolled to 0 when navigating

Reply:
My thought was that it's better to be explicit that there **_is_** an issue if you remove this piece of code, rather than "makes sure pages are scrolled to 0", because you'd expect that pages automatically do that upon navigation, but they don't, which is an issue... less guessing for the future reader.

WDYT